embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/Embrace.java

```diff
@@ -70,21 +70,36 @@
     @Override
     public void start(@NonNull Context context) {
         if (verifyNonNullParameters("start", context)) {
-            start(context, true, AppFramework.NATIVE);
+            start(context, AppFramework.NATIVE);
         }
     }
 
     @Override
+    public void start(@NonNull Context context, @NonNull AppFramework appFramework) {
+        if (verifyNonNullParameters("start", context, appFramework)) {
+            impl.start(context, appFramework);
+        }
+    }
+
+    /**
+     * @deprecated Use {@link #start(Context)} instead.
```

Comment:
I would mention that the `isDevMode` parameter does nothing now so folks can feel safe about changing this